sdk/communication/Azure.Communication.NetworkTraversal/src/CommunicationRelayClient.cs

```diff
@@ -111,14 +112,15 @@ public virtual Response<CommunicationRelayConfiguration> GetRelayConfiguration(C
 
         /// <summary>Asynchronously gets a Relay Configuration for a <see cref="CommunicationUserIdentifier"/>.</summary>
         /// <param name="communicationUser">The <see cref="CommunicationUserIdentifier"/> for whom to issue a token.</param>
+        /// <param name="routeType"> The specified <see cref="RouteType"/> for the relay request </param>
         /// <param name="cancellationToken">The cancellation token to use.</param>
-        public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(CommunicationUserIdentifier communicationUser = null, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(CommunicationUserIdentifier communicationUser = null, RouteType? routeType = null, CancellationToken cancellationToken = default)
```

Comment:
Should we consider separate overloads?
Given that we no longer need to supply a user eventually, these could become two methods
```c#
public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(CancellationToken cancellationToken = default);

public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(GetRelayConfigurationOptions options = null, CancellationToken cancellationToken = default);
```
with `GetRelayConfigurationOptions` then including `communicationUser` and `routeType`.

Reply:
@DominikMe Why would we need separate overloads? All arguments can be optional, so one overload achieves it, I think.

Reply:
Also, should I create a GetRelayConfigurationOptions? And what should it contain?

Reply:
> Also, should I create a GetRelayConfigurationOptions? And what should it contain?
Yes please. It should contain the communicationUser and routeType.

Reply:
Ok, yeah I'll do it :)

Reply:
Hey, I used the options as you suggested but we can't do (options optional and cancellationToken optional) like these:

public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(CancellationToken cancellationToken = default);

public virtual async Task<Response<CommunicationRelayConfiguration>> GetRelayConfigurationAsync(GetRelayConfigurationOptions options = null, CancellationToken cancellationToken = default);

it causes an ambiguous call if the method is called with no parameters since both overloads can take no parameters.
But after reading the documentation, it can be easily fixed by making the options not optional (ironic but makes sense haha) in that overload.